enterprise/admin/forms.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Forms to be used in the enterprise djangoapp.
+"""
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from enterprise.course_catalog_api import get_all_catalogs
+from enterprise.models import EnterpriseCustomer
+
+
+class EnterpriseCustomerAdminForm(forms.ModelForm):
+    """
+    Alternate form for the EnterpriseCustomer admin page; fetches catalog names and IDs from the Course Discovery API.
```

Comment:
Please use max column length to 79 for readability in doctstrings.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html